server/requirements.txt

```diff
@@ -1,15 +1,21 @@
+black==20.8b1
 flask==1.1.2
 flask-cors==3.0.8
 flask_restful==0.3.8
 gensim==3.8.2
+gsutil==4.59
```

Comment:
This is for deployment only. Not sure whether it should be included.

Reply:
Well I was under the impression this package would be necessary for accessing data on the google cloud instance. Is the plan to run this code on the instance itself?

Reply:
Where to run the cron update depends on the memory usage of the code. Let's keep this package for now.